src/azure-cli/azure/cli/command_modules/netappfiles/commands.py

```diff
@@ -123,7 +123,9 @@ def load_accounts_command_groups(self, netappfiles_accounts_sdk):
 
 def load_account_backup_command_groups(self, netappfiles_account_backups_sdk):
     with self.command_group('netappfiles account backup', netappfiles_account_backups_sdk) as g:
+        g.show_command('show', 'get')
         g.command('list', 'list')
+        g.command('delete', 'delete')
```

Comment:
Do you want to add `confirmation=True` for `delete` command which will give the users a chance to confirm before they delete a resource?

Reply:
I have added confirmation=True